src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -50,6 +50,11 @@ internal sealed class ExternalErrorDiagnosticUpdateSource : IDiagnosticUpdateSou
         /// </summary>
         private readonly TaskQueue _taskQueue;
 
+        /// <summary>
+        /// Task queue to serialize all the post-build work for errors reported by build.
+        /// </summary>
+        private readonly TaskQueue _postBuildTaskQueue;
```

Comment:
i think this needs much more docs.  esp. the relation to _taskQueue and how it should be used.